tests/integration/test_data/gomod_vendor_check.yaml

```diff
@@ -0,0 +1,188 @@
+# gomod package with gomod-vendor-check flag
+# repo: The URL for the upstream git repository
+# ref: A git reference at the given git repository
+# flags: A list of strings with Cachito flags
+# expected_files: Expected source files <relative_path>: <file_URL>
+# expected_deps_files: Expected dependencies files (empty)
+# response_expectations: Parts of the Cachito response to check
+# content_manifest: PURLs for image contents part
+wrong_vendor:
+  repo: https://github.com/cachito-testing/gomod-vendor-check-fail.git
+  ref:  8553df6498705b2b36614320ca0c65bc24a1d9e6
+  flags: ["gomod-vendor-check"]
+  pkg_managers: ["gomod"]
+empty_vendor:
+  repo: https://github.com/cachito-testing/gomod-vendor-check-empty-vendor.git
```

Comment:
I expected that https://github.com/cachito-testing/gomod-vendor-check-empty-vendor would have an empty `vendor/` directory. Would that be a more interesting test case than just having empty `modules.txt`? IMO that is very similar to https://github.com/cachito-testing/gomod-vendor-check-fail

Note: it can't be completely empty, since git doesn't recognize empty dirs. You would have to put something like an empty `.gitkeep` file in there

Reply:
That's exactly what I thought initially as well, but I also knew that git doesn't recognize empty dirs. I wasn't aware of the `.gitkeep` trick. I'll give it a try!

Reply:
Done!

Reply:
Nice :+1: